krux_test_utils/__init__.py

```diff
@@ -323,16 +323,16 @@ def tearDownClass(cls):
         # Remove the apps from extra_apps.
         for app_label in cls.extra_apps:
             app_name = app_label.split('.')[-1]
-            app = loading.cache.get_app(app_name)
+            app = cache.get_app(app_name)
             try:
                 # Django <= 1.6.
```

Comment:
Not that we need to change this now, but was wondering if we need to support this old of Django versions anymore? @arijitdey-sfdc

Reply:
If no other project is using this, then we can remove support for older versions.